src/main/scala/io/iohk/ethereum/network/p2p/messages/PV63.scala

```diff
@@ -38,8 +40,8 @@ object PV63 {
     }
   }
 
-  object Account {
-    implicit val rlpEndDec = new RLPEncoder[Account] with RLPDecoder[Account] {
+  object AccountImplicits {
+    implicit val rlpEncDec = new RLPEncoder[Account] with RLPDecoder[Account] {
```

Comment:
If you put these instances in the companion object of `Account` won't they be available without an explicit import? I believe that's a standard practice for typeclasses.